### PR TITLE
Add config for AWS EC2 c5d.xlarge

### PR DESCRIPTION
**Pull request by whitewum**
### What changes were proposed in this pull request?

I think an AWS EC2 config will be more comparable/illustrative for users to set conf file.

### Why are the changes needed?
These are conf file examples. I've been asked many times of recommended configs.
AWS EC2 is typical. AWS EC2 xlarge is for minimal & 12xlarge is for standard usage.

### Does this PR introduce any user-facing change?
No.

### How was this patch tested?
Not necessary.

### CODE REVIEW

**Comment by whitewum** [comment]
refer to #1831

File: etc/nebula-storage.conf.c5d.xlarge [hunk 1]
```diff
@@ -1,3 +1,9 @@
+########## bench ###########
```
Comment (dangleptr) [inline etc/nebula-storage.conf.c5d.xlarge:1]: Why rename conf.production to conf.c5d.xlarge

**Comment by darionyaphet** [comment]
Maybe we should have another config file to adapt EC2？

**Comment by whitewum** [comment]
> Maybe we should have another config file to adapt EC2？

Agree. I'll close this one.

**Pull request closed without merging**